mechanics/RELEASE-SCHEDULE.md

```diff
@@ -5,25 +5,33 @@ weight: 50
 type: "docs"
 ---
 
-Knative releases every 6 weeks. As much as possible, releases should be driven by automation, and repos should be ready to release at any point. It should also be possible to produce and consume nightly artifacts.
+Knative releases quarterly. As much as possible, releases should be driven by automation, and repos should be ready to release at any point. It should also be possible to produce and consume nightly artifacts.
```

Comment:
Can we provide/record the `crontab` here as an official decision, so people don't need to go and read through #1096 and the associated doc to figure out the pattern?

Reply:
Fixed in 5561bdd9ac744f159810d10292db49492afecea8